src/ldc/eh/common.d

```diff
@@ -14,6 +14,14 @@ import core.stdc.stdarg;
 
 import ldc.eh.fixedpool;
 
+version (ARM)
+{
+    version (iOS)
+        version = SjLj_Exceptions;
+    else
+        version = ARM_EABI_UNWINDER;
```

Comment:
I found that bit of Linux/ARM code in libgcc and libsup++, looks like it is good as is.

https://github.com/gcc-mirror/gcc/blob/master/libgcc/config/arm/unwind-arm.h#L54
```
#if (defined(linux) && !defined(__uClinux__)) || defined(__NetBSD__) \
    || defined(__FreeBSD__)
      /* Pc-relative indirect.  */
#define _GLIBCXX_OVERRIDE_TTYPE_ENCODING (DW_EH_PE_pcrel | DW_EH_PE_indirect)
```

https://github.com/gcc-mirror/gcc/blob/master/libstdc++-v3/libsupc++/eh_personality.cc#L71
```
      /* Older ARM EABI toolchains set this value incorrectly, so use a
	 hardcoded OS-specific format.  */
      info->ttype_encoding = _GLIBCXX_OVERRIDE_TTYPE_ENCODING;
```